EdgeMultiplay/Scripts/DataContracts.cs

```diff
@@ -307,21 +309,24 @@ public class CreateRoomRequest
         public int playerAvatar;
         [DataMember]
         public int maxPlayersPerRoom;
+        [DataMember (EmitDefaultValue = false)]
+        public Hashtable playerTags;
 
-        public CreateRoomRequest(string PlayerName, int PlayerAvatar, int MaxPlayersPerRoom)
+        public CreateRoomRequest(string PlayerName, int PlayerAvatar, int MaxPlayersPerRoom , Hashtable playerTags = null)
         {
             type = "CreateRoom";
             playerId = EdgeManager.gameSession.playerId;
             playerName = PlayerName;
             playerAvatar = PlayerAvatar;
             maxPlayersPerRoom = MaxPlayersPerRoom;
+            this.playerTags = playerTags;
         }
     }
 
-        /// <summary>
-        /// JoinOrCreateRoomRequest is sent to the server to join a room or create a new room if it there is no available rooms
-        /// </summary>
-        [DataContract]
+    /// <summary>
+    /// JoinOrCreateRoomRequest is sent to the server to join a room or create a new room if it there is no available rooms
```

Comment:
Nit: Should be "if there are no available rooms" or "if there is no available room".